### PR TITLE
Tooling

### PR DESCRIPTION
**Pull request by Piezoid**
### New startVEnv.sh script for starting askomics in a pyvenv.

Features:
- The CLI is similar to startServices.sh :
      - development and production modes
      - pick the `.ini` config file for the specified triplestore
- Use pyvenv (new standard module in python 3.3) for creating a virtualenv if it doesn't exists.
- Install Askomics in the VEnv as a reference to the source (pip install -e).
- In dev mode :
      - use the `pserve --reload` flag to reload on file changes,
      - enable warnings.
- In production mode :
      - Enable python "optimizations" (removing asserts and docstrings),
      - Open a browser page.

### Config files

`*.ini` files are moved to `configs/`, less clutter in the repository root.

### .gitignore

Removed unused rules in `.gitignore`. Add local `.gitignore` at `askomics/static/results/.gitignore` and `askomics/ttl/.gitignore` to keep directories indexed but ignoring theirs content.

### Travis-CI
- Test against 3.3, 3.4, 3.5.
- Simplified test requirement installation (dependencies are pulled from askomics `setup.py`)
- Enable pip cache between builds.

### Coveralls

Coveralls is web service for analyzing coverage results from Travis-CI builds. It allows to browse files and see per-statement coverage, and it compute statistics and trends of code coverage.

[Here is an example of report on my fork](https://coveralls.io/builds/6359017)

To enable it, the owner of the Askomics organization should link the account to coveralls.io (same process as for Travis-CI registration).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
   - "3.3"
+  - "3.4"
+  - "3.5"
+cache: pip
 install:
-  - pip install -r requirements.txt
+  - pip install coveralls
   - python setup.py install
-script: nosetests -v askomics --with-doctest --with-coverage --cover-package=askomics
+script: python setup.py nosetests -v --with-doctest --with-coverage --cover-package=askomics
```
Comment (abretaud) [inline .travis.yml]: Hum, shouldn't there be an && or something like that (or am I missing something obvious?)

**Comment by abretaud** [comment]
I just linked the github repo to coveralls.io, I guess we need to wait next commit/merge to make the magic work
The rest looks good I think, if you can just have a look at my comments

File: .travis.yml (hunk 1 above)
Comment (Piezoid) [inline .travis.yml]: When nosetest is installed (by default on Travis CI), `setup.py nosetests` run nosetests with the configuration specified by setup.cfg (http://nose.readthedocs.io/en/latest/api/commands.html)

File: setup.py [hunk 2]
```diff
@@ -13,6 +13,7 @@
     'pyramid_chameleon',
     'pyramid_debugtoolbar',
     'waitress',
+    'SPARQLWrapper',
```
Comment (Piezoid) [inline setup.py]: I've frozen all the dependencies in `setup.py`.

**Comment by abretaud** [comment]
Thanks! +1 for me, @ofilangi can you have a look?

**Comment by ofilangi** [comment]
+1

**Pull request merged**